oeps/best-practices/oep-0066-bp-openfeature-toggles.rst

```diff
@@ -0,0 +1,126 @@
+.. _pep_based_template:
+
+.. Below is the display in the left sidebar on RTD. Please omit leading 0's
+
+OEP-66: OpenFeature Format For Feature Toggles
+##############################################
+
+.. This OEP template is based on Python's PEP standard.
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - Link to the doc in the following format::
+
+        :doc:`OEP-0066 <oep-0066-bp-openfeature-toggles>`
+
+   * - Title
+     - Implement All Toggles Using OpenFeature Specification
+   * - Last Modified
+     - 2025-01-15
+   * - Authors
+     - Tobias Macey
+   * - Arbiter
+     - Feanil Patel <feanil@axim.org>
+   * - Status
+     - Draft
+   * - Type
+     - Best Practice
+   * - Created
+     - 2025-01-15
+   * - Review Period
+     - 2025-01-15 - 2025-02-28
+   * - Resolution
+     - <links to any discussions where the final status was decided>
+   * - References
+     - `OEP 0017 <https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0017-bp-feature-toggles.html>`_
+
+Abstract
+********
+
+OpenFeature is an open standard and ecosystem for managing feature toggles across
+various languages and frameworks. This offers a single, consistent format for managing
+the ubiquitous toggles in Open edX to improve their discoverability and maintenance.
+
+Motivation
+**********
+
+OEP 17 provided an excellent and valid argument in favor of the adoption of feature
+toggles as a core practice of the Open edX suite of software. Unfortunately, the
+implementation of that practice has led to a large variety in the methods used to manage
+those toggles, and dramatically different naming across repository boundaries. The
+adoption of the `OpenFeature
+<https://github.mit.edu/ol-platform-eng/concourse-workflow/issues/12253>`_ specification
```

Comment:
The link is private to me. Is it the same as https://openfeature.dev/specification/ ?

Reply:
Yes, that was the wrong link. Fixing now...